src/Helper/ContextHelper.php

```diff
@@ -66,6 +66,26 @@ public function isBoxedLayout(): bool
         return (bool) $this->getOption('boxed_layout');
     }
 
+    public function setLayout(string $layout): void
```

Comment:
Regardless of the layout vs menu wording question, I would prefer fail-safe methods `activateHorizontaMenu()` vs `activateVerticalMenu()` or at least `setIsHorizontalMenu(bool $isHorizontalMenu)`.

Reply:
It has been done in that way to predict the `overlap` layout:
https://preview.tabler.io/layout-navbar-overlap.html

In that case it is not only the "menu" that change, but also the `page-wrapper`.
At the end, there's not only two mode.

How do you see a better solution ?

Reply:
What speaks against a third `activateOverlapMenu()` ? I'd try to avoid leaking internal information. How we store the layout information in that helper shouldn't be a problem of the user (dev).

Reply:
Overlap layout look like horizontal layout but in code it is not.
It overlap (lol) inside the `page-wrapper`. 

By that I mean, the way of thinking for us (bundle devs) is not simply a vertical layout WITH an overlap mode.
It should be new/different layout Overlap.

Reply:
Still trying to figure it out what is the best way..
I can't get over a third layout.

## Questioning 
With what we are exchanging: 
- Layout Vertical
- Layout Horizontal 
    -> Overlap

If we are going in that way, dev can activate option `overlap` from tabler.yaml but without any changes when vertical  layout is defined. 
It could be disturbing and/or weird to have access to an option in only certain case/mode.

## Suggest
The best way for me is:
- Layout Vertical
- Layout Horizontal
- Layout Overlap (Horizontal)

## For the future
At the end, 
if we are including all templates that Tabler provides, 
we will still have options available in certain case Like "Transparent" only linked to "Vertical Layout".

If I try to visualize it as "TreeBuilder", it can look like:
- Layout Vertical
    -> Transparent [option of vertical]
  => Right
- Layout Horizontal
    -> Dark [option of horizontal]
  => Sticky
  => Condensed
    -> Dark [option of horizontal -> condensed]
- Layout Overlap (Horizontal like)

In that way, Overlap has no option available which is the case in Tabler